src/subscription_manager/gui/data/glade/factsdialog.glade

```diff
@@ -4,7 +4,7 @@
   <!--
  interface-naming-policy project-wide -->
   <object class="GtkWindow" id="system_facts_dialog">
-    <property name="width_request">500</property>
+    <property name="width_request">550</property>
```

Comment:
Just in case the translations take up too much width, I think we should wrap the `system_id_label` label.